dso-l1/src/main/java/com/tc/object/ExceptionUtils.java

```diff
@@ -46,9 +46,8 @@ public static EntityException addLocalStackTraceToEntityException(EntityID eid,
     } else if(e instanceof EntityBusyException) {
       wrappedException = new EntityBusyException(entityType, enityName, e);
     } else {
-//  just return the remote exception with remote stack, don't want to hide the type of the 
-//  exception in these cases.
-      return e;
+      // TODO we can't just return the exception as this would hide some of the stack, it must be wrapped in a new EntityException type
```

Comment:
This change worries me a bit.  I believe some code relies on the type exposed here.

Reply:
This may break some code, and more probably some tests. I nevertheless believe this is a good change, but outside the scope of this PR so I'm going to revert it.